src/SimplePanel.tsx

```diff
@@ -85,7 +86,7 @@ export const SimplePanel: React.FC<Props> = ({ options, data, width, height, tim
   const setGroupedOptionHandler = (label: string | undefined) => {
     if (label !== undefined) {
       setGroupedOption(label);
```

Comment:
Strictly speaking were missing `setMetricOption('-');` here, becuase this is doesn´t work with the MetricOption set. Since this is not the final approach it´s not that important for now.